Documentation update for Signal Messenger integration for adding URL attachment support (#20762)
home-assistant/home-assistant.github.io: source/_integrations/signal_messenger.markdown

      attachments:
        - "/tmp/surveillance_camera.jpg"
```

### Text message with an attachment from a URL

> 

Reviewer: ```suggestion

Note there is a 50mb size limit for attachments retrieved via URLs. You can set `verifyUrls` to `False` to ignore SSL errors (default `True`)

```

```yaml
...
action:
  service: notify.NOTIFIER_NAME
  data:
    message: "Person detected on Front Camera!"
    data:

>       urls:

Reviewer: ```suggestion
            verifyUrls: false
            urls:
```

      attachments:
        - "/tmp/surveillance_camera.jpg"
```

### Text message with an attachment from a URL

> Note there is a 50mb size limit for attachments retrieved via URLs. You can set `verifySsl` to `False` to ignore SSL errors (default `True`)

Reviewer: ```suggestion
To attach files from outside of HomeAssistant, the URLs must be added to the [allowlist_external_urls](https://www.home-assistant.io/docs/configuration/basic/#allowlist_external_urls) list.

Note there is a 50mb size limit for attachments retrieved via URLs. You can also set `verify_ssl` to `False` to ignore SSL errors (default `True`)
```

> Note there is a 50mb size limit for attachments retrieved via URLs. You can set `verifyUrls` to `False` to ignore SSL errors (default `True`)

Reviewer: ```suggestion
Note there is a 50mb size limit for attachments retrieved via URLs. You can set `verifySsl` to `False` to ignore SSL errors (default `True`)
```

```yaml
...
action:
  service: notify.NOTIFIER_NAME
  data:
    message: "Person detected on Front Camera!"
    data:

>       verifySsl: False

Reviewer 2: `verifySsl` isn't following our standards; it should be:

```suggestion
      verify_ssl: false
```

Additionally, I'm confused why Home Assistant needs to verify SSL, to begin with?

>             verifyUrls: false
>             urls:

Reviewer: ```suggestion
      verifySsl: False
      urls:
```

      attachments:
        - "/tmp/surveillance_camera.jpg"
```

### Text message with an attachment from a URL

> To attach files from outside of HomeAssistant, the URLs must be added to the [allowlist_external_urls](/docs/configuration/basic/#allowlist_external_urls) list.

Reviewer 2: ```suggestion
To attach files from outside of HomeAssistant, the URLs must be added to the [`allowlist_external_urls`](/docs/configuration/basic/#allowlist_external_urls) list.
```

> To attach files from outside of HomeAssistant, the URLs must be added to the [allowlist_external_urls](https://www.home-assistant.io/docs/configuration/basic/#allowlist_external_urls) list.

Reviewer 2: ```suggestion
To attach files from outside of HomeAssistant, the URLs must be added to the [allowlist_external_urls](/docs/configuration/basic/#allowlist_external_urls) list.
```

> Note there is a 50mb size limit for attachments retrieved via URLs. You can also set `verify_ssl` to `False` to ignore SSL errors (default `True`)

Reviewer 2: ```suggestion
Note there is a 50MB size limit for attachments retrieved via URLs. You can also set `verify_ssl` to `false` to ignore SSL errors (default `true`).
```
